app/models/schemas/register_new_version.rb

```diff
@@ -10,14 +10,16 @@ class RegisterNewVersion
     attr_reader :subject_name, :json
     attr_accessor :schema
     private_attr_accessor :retried
+    private_attr_reader :options
 
-    def self.call(subject_name, json)
```

Comment:
Not new but is it worth pulling in the [procto](https://github.com/snusnu/procto) gem to eliminate some of this boilerplate?